.dependency_decisions.yml

```diff
@@ -0,0 +1,68 @@
+---
+- - :whitelist
+  - MIT
+  - :who: 3scale Engineering
+    :why: 
+    :versions: []
+    :when: 2016-08-16 09:25:15.635515271 Z
+- - :whitelist
+  - Apache 2.0
+  - :who: 3scale Engineering
+    :why: 
+    :versions: []
+    :when: 2016-08-16 09:26:31.814319646 Z
+- - :whitelist
+  - MIT-LICENSE
+  - :who: 3scale Engineering
+    :why: It's the same as MIT
+    :versions: []
+    :when: 2016-08-16 09:27:49.047135842 Z
+- - :license
+  - rake
+  - MIT
+  - :who: 3scale Engineering
+    :why: Rake uses MIT but it's not automatically detected
+    :versions: []
+    :when: 2016-08-18 15:16:38.712225384 Z
+- - :whitelist
+  - BSD
+  - :who: Jeff Kaufmann and Richard Fontana (Red Hat Legal)
+    :why: 
+    :versions: []
+    :when: 2016-11-23 11:02:51.564502000 Z
+- - :whitelist
+  - New BSD
+  - :who: Jeff Kaufmann and Richard Fontana (Red Hat Legal)
+    :why: 
+    :versions: []
+    :when: 2016-11-23 11:03:04.247155000 Z
+- - :whitelist
+  - ruby
+  - :who: Jeff Kaufmann and Richard Fontana (Red Hat Legal)
+    :why: 
+    :versions: []
+    :when: 2016-11-23 11:03:15.526798000 Z
+- - :whitelist
+  - Simplified BSD
+  - :who: Jeff Kaufmann and Richard Fontana (Red Hat Legal)
+    :why: 
+    :versions: []
+    :when: 2016-11-23 11:03:33.341497000 Z
+- - :whitelist
+  - LGPLv2+
+  - :who: Jeff Kaufmann and Richard Fontana (Red Hat Legal)
+    :why: 
+    :versions: []
+    :when: 2016-11-23 11:04:13.584885000 Z
+- - :whitelist
+  - 2-clause BSDL
+  - :who: Jeff Kaufmann and Richard Fontana (Red Hat Legal)
+    :why: 
+    :versions: []
+    :when: 2016-11-23 11:04:37.776089000 Z
+- - :approve
+  - colored
```

Comment:
This is also approved, but why and by who? What is the license?

Reply:
It does have it in the source. https://github.com/defunkt/colored/blob/master/LICENSE
Would be good to link to the source and say what license it is.

Reply:
https://github.com/defunkt/colored/pull/10

almost 4 years old PR